subsys/ipc/rpmsg_service.c

```diff
@@ -105,6 +105,29 @@ static int rpmsg_service_init(const struct device *dev)
 		return err;
 	}
 
+#if !IPC_MASTER
```

Comment:
Any reason to not use  OPENAMP_MASTER config here?

Reply:
The problem with `OPENAMP_MASTER` is that it may be enabled alongside `OPENAMP_SLAVE`. However, IPC services is implemented in a way that it may either master or slave. Not both.

Reply:
Right! 
In this case It would be nice to add some controls to make sure that the IPC and OpenAMP configurations match ( or i missed the check?).

Reply:
It's in IPC Service Kconfig: https://github.com/zephyrproject-rtos/zephyr/blob/0f3eb49f3b78a376e9513a6f1107414e4d6c24d8/subsys/ipc/Kconfig#L74